media-libs/raylib/raylib-5.0.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="A simple and easy-to-use library to enjoy videogames programming"
+DESCRIPTION="Raylib is highly inspired by Borland BGI graphics \
+lib and by XNA framework and it's especially well suited for \
+prototyping, tooling, graphical applications, embedded systems and education."
+HOMEPAGE="https://github.com/raysan5/raylib"
+COPYRIGHT="2013-2024 Ramon Santamaria"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://github.com/raysan5/raylib/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="98f049b9ea2a9c40a14e4e543eeea1a7ec3090ebdcd329c4ca2cf98bc9793482"
+SOURCE_FILENAME="$portVersion.tar.gz"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	raylib = $portVersion
+	devel:libraylib = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	glfw$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	glfw${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:cmake
+	cmd:make
+	"
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+
+	cmake .. -DUSE_EXTERNAL_GLFW=ON \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_PREFIX:PATH="$prefix"
+	make
+}
+
+INSTALL()
+{
+	cd build
+	mkdir -p $libDir $includeDir
+	make install
+
+	prepareInstalledDevelLibs \
+		libraylib
+
+	oldLibDir=$libDir
+	libDir=$prefix/lib
```

Comment:
`$libDir` is a haikuporter-specific variable, so these assignments do nothing and can be removed.

Moreover, libraries should generally be installed to `$libDir` and not `$prefix/lib`, which I suppose this is doing. These are different, when a secondary architecture (i.e. x86) is used. Can the library installation path be specified via a CMake option? If the project uses [GNUInstallDirs](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html), this can be done via `$cmakeDirArgs`. If not, the library could be moved manually like you did with the headers below (before calling `prepareInstalledDevelLib`).

Reply:
Having looked at the CMake configuration files, I think using `$cmakeDirArgs` should work. Then the part moving the headers below can be removed as well.

Reply:
was able to apply the `$cmakeDirArgs` suggestion. thanks!